packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -219,17 +219,16 @@ async function getAuthHandler(module) {
   let { version } = fs.readJSONSync(packageJsonPath)
 
   if (!isInstalled(module)) {
-    const { stdout } = await execa.command(
-      `yarn npm info ${module} --fields versions --json`
-    )
-
     // If the version includes a plus, like '4.0.0-rc.428+dd79f1726'
     // (all @canary, @next, and @rc packages do), get rid of everything after the plus.
     if (version.includes('+')) {
       version = version.split('+')[0]
     }
 
-    const versionIsPublished = JSON.parse(stdout).versions.includes(version)
+    const packumentP = await fetch(`https://registry.npmjs.org/${module}`)
+    const packument = await packumentP.json()
```

Comment:
Maybe also consider wrapping it all in `try ... catch` since both `fetch` and `.json()` can throw. (Unless there already is a try/catch block further up in the call stack that should handle it)

Reply:
Done in https://github.com/redwoodjs/redwood/pull/9975/commits/d72106147b44da7c7f1356afae7cc2cc4fe07e3c. There is a top-level CLI handler but we can give a bit more context as to what threw